INTRODUCTION.md

```diff
@@ -0,0 +1,59 @@
+# X-Road Documentation
+
+## What is X-Road
+
+X-Road® is open-source software and ecosystem solution that provides unified and secure data exchange between
+organisations.
+
+![X-Road overview](X-Road_overview.png)
```

Comment:
The image should be updated to the latest version in `develop`:

https://github.com/nordic-institute/X-Road/blob/develop/X-Road_overview.png

Reply:
Merged develop into the branch so that the latest changes are now available.